viewer/viewer-awt/src/main/java/org/icepdf/ri/common/utility/search/SearchTextTask.java

```diff
@@ -98,11 +98,11 @@ private SearchTextTask(Builder builder) {
         lengthOfTask = controller.getDocument().getNumberOfPages();
 
         // setup searching format format.
-        this.searchPanel = builder.searchPanel;
-        if (searchPanel != null) {
-            searchingMessageForm = searchPanel.setupSearchingMessageForm();
-            searchResultMessageForm = searchPanel.setupSearchResultMessageForm();
-            searchCompletionMessageForm = searchPanel.setupSearchCompletionMessageForm();
+        this.searchComponent = builder.searchPanel;
```

Comment:
I find this awkward, we don't have Component or a Panel, which could lead to a mistake using the API.    Leaning toward 'baseSearchModel' a bit more now, I don't think it falls into controller territory.